### PR TITLE
Add a declared license mapping for "Android Software Development Kit License"

### PR DESCRIPTION
**Pull request by MarcelBochtler**

### CODE REVIEW

File: spdx-utils/src/main/resources/declared-license-mapping.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@
 "Affero General Public License (AGPL) v. 3": AGPL-3.0-only
 "Aladdin Free Public License (AFPL)": Aladdin
 "Amazon Software License": LicenseRef-scancode-amazon-sl
+"Android Software Development Kit License": LicenseRef-scancode-android-sdk-license
```
Comment (fviernau) [inline spdx-utils/src/main/resources/declared-license-mapping.yml:22]: Could you please provide a link in the commit message to the POM file where this was found in?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: spdx-utils/src/main/resources/declared-license-mapping.yml (hunk 1 above)
Comment (MarcelBochtler) [reply spdx-utils/src/main/resources/declared-license-mapping.yml:22]: This was intentional, as the components that use this license are not open source and therefore do not host the POM.
I could however link to a deployed POM like https://maven.google.com/com/google/android/gms/play-services-base/17.5.0/play-services-base-17.5.0.pom
Would you prefer this?

Comment (sschuberth) [reply spdx-utils/src/main/resources/declared-license-mapping.yml:22]: That's what @fviernau means: Add a link to the POM which contains "Android Software Development Kit License" as a declared license, not a link to the package using that POM. Adding the link is good practice for us to verify that the mentioned declared license really exists.

Comment (MarcelBochtler) [reply spdx-utils/src/main/resources/declared-license-mapping.yml:22]: Done

**Review by sschuberth — APPROVED** [review]

**Comment by MarcelBochtler** [comment]
i'm quite sure that this test failure is unrelated to this change.
@fviernau, @sschuberth can we merge this?

**Comment by sschuberth** [comment]
Merging despite the unrelated [Travis test failure](https://travis-ci.com/github/oss-review-toolkit/ort/builds/197381884#L4400).

**Pull request merged**